src/connection/coniql.ts

```diff
@@ -317,10 +348,12 @@ export class ConiqlPlugin implements Connection {
 
   public connect(
     connectionCallback: ConnectionChangedCallback,
-    valueCallback: ValueChangedCallback
+    valueCallback: ValueChangedCallback,
+    deviceQueried: DeviceQueriedCallback
```

Comment:
I dislike having to add an extra function here to handle the device query but wasn't sure there was a better way of doing it. Using the second function to handle device values as well seemed a bit opaque to me.

Reply:
This looks ok to me.